### PR TITLE
Letter 's' from #436

### PR DESCRIPTION
**Pull request by luzpaz**
Continuing to separate ~#428~ #436 in to separate PRs for each letter. Also removed/corrected some entries that were not relevant/precise and augmented others. This PR has been sorted as well. 

Please review closely, some of the entries still may be incorrect or may need refinement.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10113,25 +10162,38 @@ seperator->separator
...
+seqeunce->sequence
 sequece->sequence
 sequeces->sequences
+sequeence->sequence
+sequelce->sequence
 sequencial->sequential
 sequencially->sequentially
+sequencies->sequences
+sequenzes->sequences
+sequeze->sequence
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: or "squeeze"

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -10428,41 +10564,61 @@ spported->supported
...
 srunken->shrunken
 srunkn->shrunken
 stablility->stability
+stach->stack
+stae->state
+staement->statement
 staically->statically
 stainlees->stainless
 staion->station
+stamement's->statement
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: or "statements" or "statement's"

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -10505,14 +10674,19 @@ straighforward->straightforward
 straightfoward->straightforward
 straigt->straight
 straigth->straight
+strangly->strange
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: or "strangely"

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -10752,6 +10964,8 @@ surrundering->surrendering
 surveilence->surveillance
 surveill->surveil
 surveyer->surveyor
+survice->survive
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: or "service"

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -10752,6 +10964,8 @@ surrundering->surrendering
...
+survices->survives
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: or "services"

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -10763,26 +10977,39 @@ suscribes->subscribes
...
 swiming->swimming
 switchs->switches
 swith->switch
 swithch->switch
+swithed->switched
+swithing->switching
+swown->shown
 swtich->switch
 swtichs->switches
+sxl->xsl
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: this seems too domain specific

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: I'll remove this then

**Pull request merged**